src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
```

Comment:
[CODE PRACTICE] - Instead of using `split` and `reduce` directly, consider using `filter` to remove any empty declarations before reducing. This will avoid the need to check if `property` and `value` exist within the reduce callback and will make the code cleaner.